### PR TITLE
Add periodic upgrade job for dropping net_raw capability

### PR DESCRIPTION
**Pull request by umohnani8**
We are diverging the capabilities to its own MC in a patch in 4.5
so adding a periodic job to make sure that the new_raw capability
is still enabled in the restricted scc if the capabilities MC exists.
And this should persist across future upgrades if the capabilities is
not deleted.

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml
Comment (vrutkovs) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:8781]:
```suggestion
      - name: UNRESOLVED_CONFIG
```

Comment (umohnani8) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: Fixed, thanks!

**Comment by umohnani8** [comment]
This is ready, @vrutkovs PTAL

**Comment by vrutkovs** [comment]
/hold

Still references `quay.io/umohnani8/release:4.5-caps`

**Comment by umohnani8** [comment]
@vrutkovs this should be ready now, PTAL

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml
Comment (vrutkovs) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:8759]: lets use "latest 4.5 stable"?
```
        initial=$(ARTIFACTS)/release-initial.json
        if ! curl --fail -X GET -G --location https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4-stable/latest --data-urlencode 'in=>4.5.0-0 <4.6.0-0' > ${initial}; then
          echo "error: Could not find a stable version"
          exit 1
        fi
        from=$( python -c 'from __future__ import print_function; import json,sys; print(json.load(open(sys.argv[1], "r"))["name"])' "${initial}" )
```

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml
Comment (vrutkovs) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:8732]:
```suggestion
  name: release-openshift-origin-installer-e2e-aws-upgrade-4.5-stable-to-4.5-with-caps-patch
```

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml
Comment (umohnani8) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:8759]: We have to 4.5.15 or before here because the patch we want to test with the upgrade is available from 4.5.16.

Comment (vrutkovs) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:8759]: Oh, I see

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml
Comment (vrutkovs) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:8732]: That can be fixed later

**Review by vrutkovs — APPROVED** [review]
/lgtm
/hold cancel

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml
Comment (umohnani8) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: Fixed

**Comment by umohnani8** [comment]
@vrutkovs sorry for the many pings, but this should be finally ready now. Need a lgtm and approve here please :)

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Review by droslean — APPROVED** [review]

**Pull request merged**
